crates/wallet/tests/psbt.rs

```diff
@@ -171,10 +171,10 @@ fn test_psbt_multiple_internalkey_signers() {
     let mut builder = wallet.build_tx();
     builder.add_recipient(send_to.script_pubkey(), 10_000);
     let mut psbt = builder.finish().unwrap();
-    // Adds a signer for the wrong internal key, bdk should not use this key to sign
+    // Adds a signer for the wrong internal key, bdk wallet should not use this key to sign
     wallet.add_signer(
         KeychainKind::External,
-        // A signerordering lower than 100, bdk will use this signer first
+        // A signerordering lower than 100, bdk wallet will use this signer first
```

Comment:
```suggestion
        // A signerordering lower than 100, bdk_wallet will use this signer first
```
You are referring to the crate right?
Or if you are referring to the module:
```suggestion
        // A signerordering lower than 100, bdk's wallet will use this signer first
```

Reply:
This does read better as you suggested, fixed.